### ISSUE TITLE
onhoverColor cannot be specified individually?

### ISSUE THREAD

**Issue by dylanh724** [jonmiles/bootstrap-treeview#174]
If we can specify the bg color of others, of course the hover color should be different. How can I individually change the **onhoverColor**?

**Comment by jonmiles** [comment]
You're right onhoverColor can only be set globally, I will look to add this as node property as well for a greater flexibility.  Thanks
